### ISSUE TITLE
CUDAExecutionProvider is not in available provider names. Available providers: 'AzureExecutionProvider, CPUExecutionProvider'

### ISSUE THREAD

**Issue by illtellyoulater** [ASLP-lab/DiffRhythm#84]
I know this is just a warning. The generation works, but isn't the warning saying that the inference is happening on the CPU rather than on the GPU?

Full output:

```console
onnxruntime_inference_collection.py:115: UserWarning: Specified provider 'CUDAExecutionProvider' is not in available provider names.Available providers: 'AzureExecutionProvider, CPUExecutionProvider
weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
The argument `trust_remote_code` is to be used with Auto classes. It has no effect here and is ignored.
inference cost 56.69 seconds
`

**Comment by NZqian** [comment]
try to uninstall onnxruntime and install onnxruntime-gpu instead

**Comment by illtellyoulater** [comment]
Thanks. The error about "CUDAExecutionProvider" has gone, the output is now:

```
(/home/user/conda_envs/diffrhythm) user@ubuntu:/home/user/apps/DiffRhythm/scripts$ ./infer_prompt_ref.sh

Current working directory: /home/user/apps/DiffRhythm/
./g2p/sources/g2p_chinese_model/poly_bert_model.onnx

/home/user/conda_envs/diffrhythm/lib/python3.10/site-packages/torch/nn/utils/weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)

The argument `trust_remote_code` is to be used with Auto classes. It has no effect here and is ignored.
inference cost 22.12 seconds
```

So I guess everything is good now, only non important warnings are left, right?

But is it normal that the Chinese model is being used?

**Comment by NZqian** [comment]
If the input is in English, the Chinese G2P will not be used. It's the log for loading the model I think
